src/app/_hooks/useCategory.ts

```diff
@@ -12,14 +17,15 @@ export type UseCategoryProps<Entry extends Object> = {
   searchParams: NextServerSearchParams
   entries: Array<Entry>
   validCategoryIds: Array<CategoryId>
```

Comment:
We most likely don't need to pass `validCategoryIds` anymore now we pass `categoryOptions`.

We can derive the validIds from the options. What do you think?

Reply:
✅